src/app/layout.tsx

```diff
@@ -13,8 +13,8 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Agent Registration Service",
+  description: "Agent Registration Service",
```

Comment:
Thanks for tweaking the metadata! Let's make a better description to have the OG previews more engaging!